README.md

```diff
@@ -110,6 +110,41 @@ possible by the use of the `UserService` service provided by
 You can see the details in
 [`packages/shopping/src/controllers/user.controller.ts`](https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/controllers/user.controller.ts).
 
+### Authorization
+
+For an authorization demo in this example, all requests to the endpoint
```

Comment:
I personally find it a bit confusing to consider it as a demo, below is my suggested change, feel free to take it as appropriate. 
From:
`For an authorization demo in this example` 
to:
`To see authorization in action in this example`

Reply:
agreed